### ISSUE TITLE
Pruebas ya diseñadas con VGG16

### ISSUE THREAD

**Issue by NuriaVP** [NuriaVP/Transfer-Learning-EDM#12]
Importar datos de experimentos ya probados con la red de Transfer Learning VGG16, para entender su funcionamiento y, posteriormente, poder emplearla con nuestros datos de EMD.

**Comment by NuriaVP** [comment]
Incluiré el código de las prueba con VGG16 realizada cogiendo el código de: https://towardsdatascience.com/transfer-learning-with-vgg16-and-keras-50ea161580b4.

**Comment by NuriaVP** [comment]
He encontrado una forma más simplificada de usar VGG16 que además incluye cómo interperetar los resultado con matrices de confusión y curvas ROC. Lo único que no vienen datos para probarlo, debo adaptar los míos.

https://www.enmilocalfunciona.io/deep-learning-basico-con-keras-parte-3-vgg/

**Comment by NuriaVP** [comment]
También, vamos a probar el ejemplo que se describe en la siguiente web dónde se importan imágenes del experimento Image Net.

https://unipython.com/como-cargar-y-utilizar-un-modelo-de-reconocimiento-de-objetos-preconfigurado/

**Comment by NuriaVP** [comment]
**Resultados prueba VGG16 imágenes de flores**

Epoch 1/50
65/65 [==============================] - 100s 2s/step - loss: 1.7899 - accuracy: 0.4813 - val_loss: 1.1998 - val_accuracy: 0.5389
Epoch 2/50
65/65 [==============================] - 96s 1s/step - loss: 0.9098 - accuracy: 0.7012 - val_loss: 1.2066 - val_accuracy: 0.5311
Epoch 3/50
65/65 [==============================] - 97s 2s/step - loss: 0.6878 - accuracy: 0.7655 - val_loss: 1.0042 - val_accuracy: 0.6712
Epoch 4/50
65/65 [==============================] - 102s 2s/step - loss: 0.4897 - accuracy: 0.8190 - val_loss: 1.0202 - val_accuracy: 0.7004
Epoch 5/50
65/65 [==============================] - 98s 2s/step - loss: 0.3492 - accuracy: 0.8720 - val_loss: 1.0528 - val_accuracy: 0.7004
Epoch 6/50
65/65 [==============================] - 99s 2s/step - loss: 0.2521 - accuracy: 0.9022 - val_loss: 1.2829 - val_accuracy: 0.6809
Epoch 7/50
65/65 [==============================] - 104s 2s/step - loss: 0.2048 - accuracy: 0.9236 - val_loss: 1.3340 - val_accuracy: 0.6770
Epoch 8/50
65/65 [==============================] - 102s 2s/step - loss: 0.1668 - accuracy: 0.9343 - val_loss: 1.2898 - val_accuracy: 0.7043
Epoch 9/50
65/65 [==============================] - 96s 1s/step - loss: 0.1335 - accuracy: 0.9528 - val_loss: 1.3688 - val_accuracy: 0.7082
Epoch 10/50
65/65 [==============================] - 91s 1s/step - loss: 0.1307 - accuracy: 0.9528 - val_loss: 1.6204 - val_accuracy: 0.7082
Epoch 11/50
65/65 [==============================] - 90s 1s/step - loss: 0.0959 - accuracy: 0.9640 - val_loss: 1.3645 - val_accuracy: 0.7082
Epoch 12/50
65/65 [==============================] - 90s 1s/step - loss: 0.0707 - accuracy: 0.9771 - val_loss: 1.4691 - val_accuracy: 0.7160
Epoch 13/50
65/65 [==============================] - 91s 1s/step - loss: 0.0639 - accuracy: 0.9757 - val_loss: 1.3684 - val_accuracy: 0.7140
Epoch 14/50
65/65 [==============================] - 90s 1s/step - loss: 0.0527 - accuracy: 0.9800 - val_loss: 1.4814 - val_accuracy: 0.7043
Epoch 15/50
65/65 [==============================] - 90s 1s/step - loss: 0.0473 - accuracy: 0.9820 - val_loss: 1.6460 - val_accuracy: 0.7062
Epoch 16/50
65/65 [==============================] - 90s 1s/step - loss: 0.0446 - accuracy: 0.9839 - val_loss: 1.5233 - val_accuracy: 0.6984
Epoch 17/50
65/65 [==============================] - 90s 1s/step - loss: 0.0480 - accuracy: 0.9830 - val_loss: 1.5841 - val_accuracy: 0.7218
Epoch 18/50
65/65 [==============================] - 90s 1s/step - loss: 0.0458 - accuracy: 0.9854 - val_loss: 1.6690 - val_accuracy: 0.7004
Epoch 19/50
65/65 [==============================] - 91s 1s/step - loss: 0.0433 - accuracy: 0.9839 - val_loss: 1.6735 - val_accuracy: 0.7179
Epoch 20/50
65/65 [==============================] - 90s 1s/step - loss: 0.0360 - accuracy: 0.9888 - val_loss: 1.6850 - val_accuracy: 0.7179
Epoch 21/50
65/65 [==============================] - 90s 1s/step - loss: 0.0296 - accuracy: 0.9908 - val_loss: 1.7082 - val_accuracy: 0.7179
Epoch 22/50
65/65 [==============================] - 90s 1s/step - loss: 0.0213 - accuracy: 0.9912 - val_loss: 1.8468 - val_accuracy: 0.7198

**Comment by NuriaVP** [comment]
Tarea realizada en el commit: _3ebf8f7146d22b1a032df7f74b65ef3ce07c1079_

**Comment by NuriaVP** [comment]
Como ya comente anteriormente...

> He encontrado una forma más simplificada de usar VGG16 que además incluye cómo interperetar los resultado con matrices de confusión y curvas ROC. Lo único que no vienen datos para probarlo, debo adaptar los míos.
> 
> https://www.enmilocalfunciona.io/deep-learning-basico-con-keras-parte-3-vgg/

Pues bien, he repetido este experimento en el commit: _ba1246bf1bee5febeabd71d7b798c91f05947f2d._

No adjunto los resultados porque debido a los tiempos de ejecución, solo he probado con los epochs.
